File: planit v0.2/Source/HotelsSource/Objects/HDKCity+Hotellook.m
#import "planit_v0_2-Swift.h"
#import "HDKCity+Hotellook.h"
#import "HLPoiManager.h"

@implementation HDKCity (HL)

- (NSArray<HDKLocationPoint *> *)airports
{
    return [HLPoiManager filterPoints:self.points byCategories:@[HDKLocationPointCategory.kAirport]];
}

@end

File: planit v0.2/HotelsSource/Objects/HDKCity+Hotellook.swift
//  Converted with Swiftify v1.0.6402 - https://objectivec2swift.com/
import HotellookSDK

extension HDKCity {
    func airports() -> [HDKLocationPoint] {
        return HLPoiManager.filterPoints(points(), byCategories: [HDKLocationPointCategory.kAirport])
    }
}